Formula/ascheck.rb

```diff
@@ -0,0 +1,19 @@
+class Ascheck < Formula
+  desc "Command-line app that checks all of your apps for their Apple Silicon support"
+  homepage "https://github.com/harryzcy/ascheck"
+  url "https://github.com/harryzcy/ascheck/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "5a90fa2b521ca68fced0866a8055c36fb7367a37bd8d04ac5e3ad391b6706fa4"
+  license "MIT"
+  head "https://github.com/harryzcy/ascheck.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", "ascheck"
+    bin.install "ascheck"
+  end
+
+  test do
+    system "#{bin}/ascheck", "--help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Sure, thank you.